.ci/watchdog/Watchdog.py

```diff
@@ -135,6 +140,34 @@ def _read_config_file(self):
             data = {_PR_REPORTS_CONFIG_KEY: {}}
         return data
 
+    @staticmethod
+    def _should_ignore(pr, criteria):
+        """
+        Determine if PR should be ignored based on criteria in ignore.json file.
+
+            :param pr:          Single PR being currently checked
+            :param criteria:    Dictionary containing ignore criteria
+            :type pr:           github.PullRequest.PullRequest
+            :type criteria:     Dict
+
+            :return:            Returns True if PR should be ignored
+            :rtype:             Bool
+        """
+        
+        # Go through ignore criteria and perform logical operations defined in ignore.json config file
+        try:
+            for criterion in criteria['ignore_if']:
+                for value in criterion.get('values'):
+                    expression = '{} {} \"{}\"'.format(criterion.get('variable_name'), criterion.get('operation'), value)
+                    if eval(expression):
```

Comment:
This is a very dangerous construction. You I can provide any instructions I want in your `json` file. Consider a different approach rather then using `eval`

Reply:
Changed approach. Ignore criteria stated explicitly in method instead of being read from file (which is even more readable as long as number of these filters is low).

Reply:
Fixed. Not reading ignore criteria from a file anymore.